webviz_ert/views/ensemble_selector_view.py

```diff
@@ -29,11 +29,13 @@ def ensemble_selector_list(parent: WebvizPluginABC) -> List[Component]:
                     dbc.Col(
                         html.Button(
                             id=parent.uuid(f"parameter-selector-button"),
-                            children=("Toggle selector visibilities"),
+                            children=("Hide Selectors"),
```

Comment:
Why change name ? I guess the button actually `toggles` and not only `hide` the selectors?

Reply:
`Hide Selectors` is only the initial name - logic was added to switch the name between `hide` and `show`, and the reason for removing the word visibilities is a visual one - the button becomes smaller